spec/spec_helper.rb

```diff
@@ -9,7 +9,7 @@
 require 'rspec/rails'
 
 require "manageiq/password/rspec_matchers"
-ManageIQ::Password.v2_key = ManageIQ::Password::Key.new
+ManageIQ::Password.key = ManageIQ::Password::Key.new
```

Comment:
Is this here because we want to run without a `manageiq/certs/v2_key?
I guess this would run without a manageiq checked out at all.

Wonder if we could setup the v2 key the same way we set it up for all other processes so we would not rely upon this api at all (and we could be loose with the dependency)

Reply:
This is because schema runs in the context of a dummy app, so something needs to set the key.

> Wonder if we could setup the v2 key the same way we set it up for all other processes so we would not rely upon this api at all 

Not sure I follow - this API _is_ the way we set up the v2 key in other processes (unless I'm missing something)

Reply:
Or are you suggesting committing a fake v2_key and then use that for test by setting the key root?

Reply:
This is the only spot that uses this code.
So there is probably another way for us to have a dummy v2_key.

Reply:
Oh that's very interesting - didn't realize `.key=` was literally only used here...I'm good with switching to `.key_root=`

Reply:
Though to be honest I actually _prefer_ this interface 😉